test/cached-build.test.js

```diff
@@ -39,7 +40,7 @@ function runBuild(distFolder, onDone) {
 }
 
 test('it should correctly generate all required pino files when the cache is enabled on Webpack', (t) => {
-  t.plan(7)
+  t.plan(9)
 
   const distFolder = resolve(__dirname, '../tmp/cached-build')
```

Comment:
why don't we use tap's testdir feature so we don't have to come up with fancy names for folders?

Reply:
Because i didn't know of it's existence :)

Thanks for pointing this out!

Reply:
done